### PR TITLE
flattened if else statements for better code structure

### PR DESCRIPTION
**Pull request by jaykay99992**
changed 
```

if:

else:
    if:

    else:
```

to 

```

if:

elif:

else:
```

for better code structure and clarity.

### CODE REVIEW

File: src/flask/cli.py
Comment (CDRouleau) [inline src/flask/cli.py:1]: Code clearer with the elsif statement.

**Review by CDRouleau — APPROVED** [review]
Good job

**Comment by jaykay99992** [comment]
@CDRouleau Just a final confirmation, do I merge the code like this ?

**Comment by CDRouleau** [comment]
> @CDRouleau Just a final confirmation, do I merge the code like this ?

yes you can merge the code like this

**Comment by jaykay9999** [comment]
Sure, code merged.

**Pull request merged**